### PR TITLE
Fix device template command 

### PR DESCRIPTION
**Pull request by mjamescompton**
#### Summary

Incorrect command shown in docs.

#### Changes

- Updated device -> end-device

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible with existing API, storage, configuration and CLI, according to the compatibility commitments in `README.md`. The target branch is set to `master` if the changes are fully compatible with existing API, database, configuration and CLI.
- [x] Testing: The changes are covered with unit tests. The changes are tested manually as well.
- [x] Documentation: Relevant documentation is added or updated.
- [x] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [x] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

**Comment by mjamescompton** [comment]
I thought this was an issue as I was having issues with templating, it is actually valid . I did notice that we are not consistent in the docs what we use for the end-devices command, above we use `end-device` and `device` in a single command and in create-end-device doc we use `end-devices`. @htdvisser Should we be more consistent with this and always use `end-devices` in the docs?

**Comment by htdvisser** [comment]
The guideline is to always use the canonical name of the command (the name that the CLI prints when running with `--help`)

File: doc/content/devices/end-device-templates/creating.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ You can use the `end-device template create` command to create a template from a
 Pipe the output from getting a device to create a template, for example:
 
 ```bash
-$ ttn-lw-cli device get test-app test-dev \
+$ ttn-lw-cli end-device get test-app test-dev \
```
Comment (htdvisser) [inline doc/content/devices/end-device-templates/creating.md:22]: The canonical name of this command is `ttn-lw-cli end-devices get`

File: doc/content/devices/end-device-templates/creating.md [hunk 2]
```diff
@@ -19,10 +19,10 @@ You can use the `end-device template create` command to create a template from a
 Pipe the output from getting a device to create a template, for example:
 
 ```bash
-$ ttn-lw-cli device get test-app test-dev \
+$ ttn-lw-cli end-devices get test-app test-dev \
   --lorawan-version \
   --lorawan-phy-version \
-  | ttn-lw-cli end-device template create > template.json
+  | ttn-lw-cli end-devices template create > template.json
```
Comment (htdvisser) [inline doc/content/devices/end-device-templates/creating.md:25]: Pretty sure it's plural "templates", but other than that fine

**Pull request merged**